src/kalenderintegration.hbs

```diff
@@ -0,0 +1,270 @@
+---
+layout: layout
+title: Kalenderintegration
+stylesheets:
+  - /static/css/kalenderintegration.css
+---
+
+{{> partials/navigation active="funktionen" }}
+
+<main class="flex-1">
+
+	<div class="w-full max-w-screen-xl mx-auto">
+		<div class="px-12 sm:px-20 xl:px-12 2xl:px-4">
+			<h1 class="text-2xl font-bold" id="zeiterfassung">
+				Kalenderintegration
+			</h1>
+
+ 			<div class="mt-12 flex flex-col md:flex-row gap-8">
+ 				<div class="flex-1 flex flex-col">
+ 					<ul class="mt-4 flex flex-col gap-3">
+ 						<li class="flex items-start">
+ 							<span class="mr-3">
+ 								{{> partials/icon-check class='w-6 h-6'}}
+ 							</span>
+ 							iCal Standard fuer mehr Flexibilitaet
+ 						</li>
+ 						<li class="flex items-start">
+ 							<span class="mr-3">
+ 								{{> partials/icon-check class='w-6 h-6'}}
+ 							</span>
+ 							Rasend schnell
+ 						</li>
+ 						<li class="flex items-start">
+ 							<span class="mr-3">
+ 								{{> partials/icon-check class='w-6 h-6'}}
+ 							</span>
+ 							DSGVO-konform
+ 						</li>
+ 						<li class="flex items-start">
+ 							<span class="mr-3">
+ 								{{> partials/icon-check class='w-6 h-6'}}
+ 							</span>
+ 							Kalender mit den eigenen Abwesenheiten
+ 						</li>
+						<li class="flex items-start">
+ 							<span class="mr-3">
+								{{> partials/icon-check class='w-6 h-6'}}
+							</span>
+							Kalender auf Abteilungsebene
+						</li>
+						<li class="flex items-start">
+ 							<span class="mr-3">
+								{{> partials/icon-check class='w-6 h-6'}}
+							</span>
+							Firmenkalender
+						</li>
+ 					</ul>
+ 				</div>
+			</div>
+		</div>
+	</div>
+
+		<h2 class="sr-only">
+  		Funktionen
+  	</h2>
+
+  	<div class="mt-16 md:mt-32 bg-gradient-to-b from-white to-sky-100">
+  		<div class="w-full max-w-screen-xl mx-auto md:px-8 2xl:px-0">
+
+				<div class="zeit-feature-card-border">
+  				<div class="zeit-feature-card">
+  					<div class="zeit-feature-card_description">
+  						<h3>
+  							iCal Standard für mehr Flexibilität
+  						</h3>
+  						<p>
+  							Alle gängigen Kalenderanwendungen unterstützen das iCalendar Datenformat und können externe Kalender integrieren.
+  						</p>
+  						<p>
+  							Einfach Link generieren und in der Anwendungen der Wahl importieren.
```

Comment:
Links und mehr Infos zu den gängigen Kalendern? wie in https://urlaubsverwaltung.cloud/hilfe/kalender/#welche-moeglichkeiten-habe-ich-eine-ical-kalenderfreigabe-in-meinem-client-zu-verwenden

Reply:
fixed